DTS-STN/dts-rmp
# 103 multiple contacts

# Description

- Call api to fetch contacts and populate dropdown, contact name should be displayed
- Display selected contact, including contact name, department, contact email
- display the last engagement title of the selected contact, with the engagement date and number of other participants

## Test Instructions

- Navigate to the add contacts & engagement page, select engagement
- play with the dropdown to see if contacts displayed properly 

## Note

- Adding multiple contact not available yet, will add it in another task

## Reviewers

@nghe0001 @MarcoGoC @bcloutier7 @will0684 @andr0272 

## Checklist
